### PR TITLE
Improve Local Tutorial (split for packages vs contributors)

### PR DESCRIPTION
**Pull request by morgo**
This PR changes the "local" tutorial to be based on packages provided by PlanetScale, simplifying a number of steps.

The compile steps are moved into a separate tutorial and placed under contributing.

There are some other small wording/grammatical/formatting changes.

Signed-off-by: Morgan Tocker <tocker@gmail.com>

### CODE REVIEW

File: content/en/docs/contributing/build-from-source.md [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+
+#### Common Build Issues
+
+{{< info >}}
+If you run into issues or have questions, we recommend posting in our [Slack channel](https://vitess.slack.com), click the Slack icon in the top right to join. This is a very active community forum and a great place to interact with other users.
+{{< /info >}}
+
+##### Python errors
+
+The end-to-end test suite currently requires Python 2.7. We are working on removing this dependency, and recommend that you bootstrap with test-building disabled (`BUILD_TESTS=0 ./bootstrap.sh`).
```
Comment (deepthi) [inline content/en/docs/contributing/build-from-source.md]: unless you are planning to contribute?

File: content/en/docs/tutorials/local.md [hunk 2]
```diff
@@ -1,275 +1,84 @@
...
-{{< info >}}
-If you run into issues or have questions, we recommend posting in our [Slack channel](https://vitess.slack.com), click the Slack icon in the top right to join. This is a very active community forum and a great place to interact with other users.
-{{< /info >}}
+This guide covers installing Vitess locally for testing purposes, from pre-compiled binaries. We will launch 3 copies of `mysqld`, so it is recommended to have greater than 4GB RAM, as well as 20GB of available disk space.
 
-## Manual Build
+## Install Packages
 
-The following sections explain the process for manually building Vitess on Linux without using Docker.
+PlanetScale provides [weekly builds](https://github.com/planetscale/vitess-releases/releases) of Vitess for 64-bit Linux.
```
Comment (deepthi) [inline content/en/docs/tutorials/local.md]: @sougou are we ok to say this on the vitess website?

**Comment by deepthi** [comment]
Another comment: for the contributors it will be good to provide instructions on running tests.

File: content/en/docs/contributing/build-from-source.md (hunk 1 above)
Comment (morgo) [reply content/en/docs/contributing/build-from-source.md]: This is the contributor guide, so you have a point :-) I will have to think about a better way to describe it.

Comment (morgo) [reply content/en/docs/contributing/build-from-source.md]: I've suggested using Docker for now. It should work for most users.

**Comment by morgo** [comment]
@deepthi PTAL again. I've also snuck in another change to s/lock service/topo service/

File: content/en/docs/overview/concepts.md [hunk 3]
```diff
@@ -91,11 +91,11 @@ There can be a different `ServedFrom` record for each `(cell, tablet type)` pair
...
 
-## Topology Service
+## Topology Service (TOPO)
 
-The [*Topology Service*](../../user-guides/topology-service) is a set of backend processes running on different servers. Those servers store topology data and provide a distributed locking service.
+The [*Topology Service*](../../user-guides/topology-service) (also referred to as the lock service) is a set of backend processes running on different servers. Those servers store topology data and provide a distributed locking service.
 
-Vitess uses a plug-in system to support various backends for storing topology data, which are assumed to provide a distributed, consistent key-value store. By default, our [local example](../../tutorials/local) uses the ZooKeeper plugin, and the [Kubernetes example](../../tutorials/kubernetes) uses etcd.
+Vitess uses a plug-in system to support various backends for storing topology data, which are assumed to provide a distributed, consistent key-value store. By default, Vitess uses the `etcd2` plugin.
```
Comment (deepthi) [inline content/en/docs/overview/concepts.md]: I would change this back to make it clear that we use etcd2 by default in the tutorials only. Vitess itself is topo-server-agnostic and is expected to work the same regardless of the choice of topo.

Comment (morgo) [reply content/en/docs/overview/concepts.md]: Thanks, I've made it clear this is the tutorials.

File: content/en/docs/tutorials/vagrant.md [hunk 4]
```diff
@@ -75,7 +75,7 @@ make site_test
 
 ## Start a Vitess cluster
 
-After completing the instructions above to [build Vitess](#build-vitess-vagrant), you can use the example scripts in the Github repo to bring up a Vitess cluster on your local machine. These scripts use ZooKeeper as the lock service. ZooKeeper is included in the Vitess distribution.
+After completing the instructions above to [build Vitess](#build-vitess-vagrant), you can use the example scripts in the Github repo to bring up a Vitess cluster on your local machine. These scripts use ZooKeeper as the [topology service](../../overview/concepts). ZooKeeper is included in the Vitess distribution.
```
Comment (deepthi) [inline content/en/docs/tutorials/vagrant.md]: Should this now say etcd2 instead of zookeeper?

Comment (morgo) [reply content/en/docs/tutorials/vagrant.md]: Fixed. I have also changed the suggestions from mailing list for help to slack.

File: content/en/docs/tutorials/local.md (hunk 2 above)
Comment (sougou) [reply content/en/docs/tutorials/local.md]: Yes. It is ok for us to accept the gracious help from PlanetScale :).

**Pull request merged**